app/views/layouts/application.html.erb

```diff
@@ -17,7 +17,7 @@
   <body>
     <div class="body-inside">
       <%= render "layouts/flash_messages" %>
-      <%= render "account/shared/navigation" %>
+      <%= render "layouts/navigation" %>
       <%= render "shared/breadcrumbs" %>
```

Comment:
![image](https://github.com/user-attachments/assets/40fa17a4-c67d-47a5-864a-b3d115040e6d)
For unsigned user style is the same.

![image](https://github.com/user-attachments/assets/eb5c4483-4d05-4a29-b271-0ca45fa36a76)
for admin